windows-driver-docs-pr/devtest/control-guid-file.md

```diff
@@ -26,7 +24,7 @@ Developers of trace providers typically supply this file. However, if you have t
 
 In the source code, find the definition of the [WPP\_CONTROL\_GUIDS](https://msdn.microsoft.com/library/windows/hardware/ff556186) macro. The GUID value and GUID friendly name are shown in bold in the example below.
 
-```
+```C
```

Comment:
Intended?

Reply:
It specifies the language as C, and enables syntax highlighting.